src/tabwidget.cpp

```diff
@@ -161,8 +161,9 @@ void TabWidget::zoomReset()
     terminalHolder()->currentTerminal()->impl()->zoomReset();
 }
 
-void TabWidget::updateTabIndices()
+void TabWidget::updateTabIndices(int, int)
```

Comment:
Adding not used parameters to slot just because the signal's signature is not needed. This:
```diff
diff --git a/src/tabwidget.cpp b/src/tabwidget.cpp
index 0935747..2f0ca15 100644
--- a/src/tabwidget.cpp
+++ b/src/tabwidget.cpp
@@ -161,7 +161,7 @@ void TabWidget::zoomReset()
     terminalHolder()->currentTerminal()->impl()->zoomReset();
 }
 
-void TabWidget::updateTabIndices(int, int)
+void TabWidget::updateTabIndices()
 {
     /* The parameters are not used, needed only to pair with the signal */
     for(int i = 0; i < count(); i++)
diff --git a/src/tabwidget.h b/src/tabwidget.h
index 15720f1..930d4ab 100644
--- a/src/tabwidget.h
+++ b/src/tabwidget.h
@@ -107,7 +107,7 @@ protected:
      */
     bool eventFilter(QObject *obj, QEvent *event);
 protected slots:
-    void updateTabIndices(int from = 0, int to = 0);
+    void updateTabIndices();
     void onTermTitleChanged(QString title, QString icon);
 
 private:
```
gets compiled correctly.